src/test/cli/rbd/not-enough-args.t

```diff
@@ -188,6 +188,7 @@
   rbd: locker was not specified
   [22]
   $ rbd bench-write
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
Interesting, I have the same Commit ID in local and it contains these fixes, but the remote one in github is different. As they have the same ID, I can't push it now with an error of "Everything is updated.". So I will --ammend and push with --force to force update it.